src/connectors/numeric-menu/__tests__/connectNumericMenu-test.ts

```diff
@@ -435,26 +472,31 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/numeric-men
       items: listOptions,
     });
 
-    const helper = jsHelper({});
+    const helper = jsHelper(createSearchClient(), '');
     helper.search = jest.fn();
 
-    widget.init({
-      helper,
-      state: helper.state,
-      createURL: () => '#',
-    });
+    widget.init!(
+      createInitOptions({
+        helper,
+        state: helper.state,
+      })
+    );
 
     const refine =
       rendering.mock.calls[rendering.mock.calls.length - 1][0].refine;
     // a user selects a value in the refinement list
     refine(encodeValue(listOptions[0].start, listOptions[0].end));
 
-    widget.render({
-      results: new SearchResults(helper.state, [{}]),
-      state: helper.state,
-      helper,
-      createURL: () => '#',
-    });
+    widget.render!(
+      createRenderOptions({
+        results: new SearchResults(helper.state, [
+          createSingleSearchResponse(),
+        ]),
+        state: helper.state,
+        helper,
+        createURL: () => '#',
```

Comment:
You can remove `createURL` when they're empty. They're covered by `createRenderOptions`.